lib/matplotlib/tests/test_axes.py

```diff
@@ -411,6 +411,107 @@ def test_polar_units():
     assert_true(isinstance(plt.gca().get_xaxis().get_major_formatter(), units.UnitDblFormatter))
 
 
+def test_polar_format_coord():
+    import matplotlib.testing.jpl_units as units
+    units.register()
+
+    deg = units.UnitDbl(1.0, "deg")
+    km = units.UnitDbl(1.0, "km")
+
+    fig = plt.figure()
+    ax = plt.subplot(111, polar=True)
+
+    # Make sure this doesn't throw any exceptions
+    ax.format_coord( 30.0*deg, 4.0*km )
+
+
+@image_comparison(baseline_images=['polar_rgrids'])
```

Comment:
As these tests add new image comparison tests, you'll also need to commit the new baseline images.